### PR TITLE
add chira galaxy tool

### PR DESCRIPTION
**Pull request by pavanvidem**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/chira/chira_merge.xml [hunk 1]
```diff
@@ -0,0 +1,97 @@
+<tool id="chira_merge" name="ChiRA merge alignments" version="1.0">
+    <description>Chimeric Read Annotator: merge aligned positions</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command><![CDATA[
+        chira_merge.py
+        -b $alignments
```
Comment (bgruening) [inline tools/chira/chira_merge.xml]: single-quote

File: tools/chira/chira_quantify.xml [hunk 2]
```diff
@@ -0,0 +1,64 @@
+<tool id="chira_quantify" name="ChiRA qauntify" version="1.0">
+    <description>Chimeric Read Annotator: quantify aligned loci to later score the alignments</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command><![CDATA[
+        chira_quantify.py
+        -b $segments
```
Comment (bgruening) [inline tools/chira/chira_quantify.xml]: single-quotes

File: tools/chira/chira_quantify.xml [hunk 3]
```diff
@@ -0,0 +1,64 @@
+<tool id="chira_quantify" name="ChiRA qauntify" version="1.0">
```
Comment (bgruening) [inline tools/chira/chira_quantify.xml]: the version should be a macro here

Comment (bgruening) [reply tools/chira/chira_quantify.xml]: @WRAPPER_VERSION@0

File: tools/chira/chira_map.xml [hunk 4]
```diff
@@ -0,0 +1,148 @@
+<tool id="chira_map" name="ChiRA map reads" version="1.0">
```
Comment (bgruening) [inline tools/chira/chira_map.xml]: @WRAPPER_VERSION@0

File: tools/chira/chira_extract.xml [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+    <description>Chimeric Read Annotator: extrat the chimeras</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="aggressive"><![CDATA[
+        #set $genomic_fasta = ''
+        #if str($annotation.annot_choice) == "yes":
+            #if str($annotation.fasta_source.fasta_source_selector) == 'history':
+                #set $genomic_fasta = $annotation.fasta_source.fasta
```
[the remark this answers is not in the document]
Comment (pavanvidem) [reply tools/chira/chira_extract.xml]: @bgruening could you please check if those single-quotes are enough?

**Pull request merged**